common/display/displayplanemanager.cpp

```diff
@@ -111,6 +111,9 @@ std::tuple<bool, DisplayPlaneStateList> DisplayPlaneManager::ValidateLayers(
   auto layer_begin = layers.begin();
   auto layer_end = layers.end();
   bool render_layers = false;
+  // Below flag takes care of condition,when the cursor layer is assigned the
+  // lower zorder
+  bool cursor_plane_enable = true;
```

Comment:
From Gen9 onwards all the planes are windowable, we can flip the cursor layer to primary also.  But in Gen8 and below the primary pane is non windowable we cant flip the cursor layer to primary.
Please move the cursor layer to cursor plane. Always cursor is expected to have highest z-order, not sure why we get some lower zorder maybe due to some transparency in the high layer things are working